docker-compose/compute_wrapper/var/db/postgres/specs/spec-multi-region.json

```diff
@@ -0,0 +1,161 @@
+{
+    "format_version": 1.0,
+
+    "timestamp": "2022-10-12T18:00:00.000Z",
```

Comment:
Should these be generated based on the time or do we fix this?

Reply:
I don't think it matters to us. This spec is for the compute_tools but I don't see it uses the timestamp for anything